lib/config.js

```diff
@@ -272,6 +272,52 @@ const conf = convict({
       format: Array,
       env: 'HPKP_PIN_SHA256'
     }
+  },
+  serverCache: {
+    redis: {
+      host: {
+        default: '127.0.0.1',
+        env: 'REDIS_HOST',
+        format: 'ipaddress',
+        doc: 'Url for redis host',
+      },
+      keyPrefix: {
+        default: 'fxa-profile',
+        env: 'REDIS_KEY_PREFIX',
+        format: String,
+        doc: 'which redis key prefix to use for hapi server cache'
+      },
+      port: {
+        default: 6379,
+        env: 'REDIS_PORT',
+        format: 'port',
+        doc: 'port for redis server'
+      }
+    },
+    emailsToCache: {
+      doc: 'Only enable email caching for addresses matching this regex.',
```

Comment:
Oh sorry it was not clear, so this regex is not just for `email caching`, but for any caching. 

If request matches ` /.+@mozilla\.com$|.+@restmail\.net$|.+@softvisioninc\.eu$|.+@softvision\.(com|ro)$/,` then CACHE request. if not, then do not cache. 

This will allow us to test caching out first and if everything is okay then roll out to everybody